### DISCUSSION TITLE
How can I archive multiple source of posts?

### DISCUSSION THREAD

**Discussion by sinnedramos08** [adityatelange/hugo-PaperMod#1536] [Q&A] [answered]
Hi, I wanted to archive every posts I make, but these posts comes from different folders, not just in one "posts" folder. Here are the folders and top menus I made. 
![image](https://github.com/adityatelange/hugo-PaperMod/assets/140233945/d34a5681-67f3-4886-9c46-7a11943db710)
![image](https://github.com/adityatelange/hugo-PaperMod/assets/140233945/40928102-4619-4357-b456-8683d08be81d)

Here are the posts in each folder/menu
![image](https://github.com/adityatelange/hugo-PaperMod/assets/140233945/d96ad757-8e84-430a-8ad9-8f76dc6ee213)
![image](https://github.com/adityatelange/hugo-PaperMod/assets/140233945/17c42271-7063-41d1-8284-b71d24fe7d16)
![image](https://github.com/adityatelange/hugo-PaperMod/assets/140233945/120b85b6-d581-414f-ac6c-7207ab405a4d)

And here is what the archives are currently recording. It only records the posts in the Blogs folder
![image](https://github.com/adityatelange/hugo-PaperMod/assets/140233945/50830da4-ec3d-469a-9d1e-8fda04a82487)

How can I make an archive  in which it can record all the posts from all the menu posts sources?

**Comment by adityatelange** [comment, marked as answer]
`ShowAllPagesInArchive`
